src/metabase/search/appdb/core.clj

```diff
@@ -23,8 +24,9 @@
 
 (set! *warn-on-reflection* true)
 
-(defmethod search.engine/supported-engine? :search.engine/fulltext [_]
-  (and (public-settings/experimental-fulltext-search-enabled)
+(defmethod search.engine/supported-engine? :search.engine/appdb [_]
+  (and (or (not config/is-prod?)
+           (= "appdb" (some-> (public-settings/search-engine) name)))
```

Comment:
I don't trust settings enforce the namespace

Reply:
not sure I understand the code and the comment at the same time :)

Reply:
I mean it would be nice for the setting to just return `:search.engine/appdb`